configs/logging/fluentd/forseti.conf

```diff
@@ -0,0 +1,24 @@
+<source>
+    @type tail
+
+    <parse>
+        @type multiline
+        format_firstline /^\[forseti\-security\]/
+        format1 /^(?<message>.*)/
```

Comment:
Once you add the time in, you need to extract it in this parser so that the log has the right timestamp in stackdriver:

https://docs.fluentd.org/v0.12/articles/parser_multiline#example